### PR TITLE
MNT: rerender

### LINKED ISSUE #11: @conda-forge-admin, please re-render
@conda-forge-admin, please re-render.

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #11.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

Fixes #11

### CODE REVIEW

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

File: .azure-pipelines/azure-pipelines-linux.yml [hunk 1]
```diff
@@ -32,6 +32,11 @@ jobs:
   timeoutInMinutes: 360
 
   steps:
+  - script: |
+         rm -rf /opt/ghc
+         df -h
+    displayName: Manage disk space
```
Comment (jakirkham) [inline .azure-pipelines/azure-pipelines-linux.yml:38]: Now with the deletion fix 😄

cc @h-vetinari @pearu

**Comment by h-vetinari** [comment]
Thanks!

**Comment by h-vetinari** [comment]
Interestingly, the merge created no CI run. Should be fine though...

**Comment by jakirkham** [comment]
Yeah this seems to happen sometimes ( https://github.com/conda-forge/conda-forge.github.io/issues/1082 ). Not sure why though

**Pull request merged**